keyboards/ducky/one2mini/readme.md

```diff
@@ -0,0 +1,15 @@
+# one2mini
+
+![one2mini](http://www.duckychannel.com.tw/page-en/One_2_Mini_RGB/images/One2_Mini_01_02.jpg)
+
+A short description of the keyboard/project
+
+Keyboard Maintainer: [/u/KeepItUnder](https://github.com/GitWellBack)  
+Hardware Supported: Ducky One 2 Mini RGB/ NUC123SD4AN0 / MBI5042GP
```

Comment:
This line needs two spaces added at the end for GitHub to render this section as intended.

Also a slight spacing adjustment for visual polish.

```suggestion
Hardware Supported: Ducky One 2 Mini RGB / NUC123SD4AN0 / MBI5042GP  
```